types/checksum.go

```diff
@@ -10,6 +10,10 @@ import (
 // The length of a checksum must always be ChecksumLen.
 type Checksum []byte
 
+func (cs Checksum) String() string {
+	return hex.EncodeToString(cs)
+}
```

Comment:
It looks like this is not used anymore. Or will this be needed when we go back to using the Checksum type in the message pack again?

Reply:
I think implementing the Stringer interface in a way to use the common way to print checksums in the CosmWasm ecosystem (lower case hex) is a good thing to have in any case. There might me use cases in wamsmd for that if not used here.

Reply:
Good point 👍